examples/03_better_tool/ex_better_tool/main.rb

```diff
@@ -0,0 +1,183 @@
+# Copyright 2021 Trimble Inc
+# Licensed under the MIT license
+
+require 'sketchup.rb'
+
+module Examples
+  module BetterTool
+
+    class LineTool
+
+      # Threshold in logical screen pixels for when the mouse is considered to
+      # be dragged.
+      APERTURE = 10
```

Comment:
I'd call this `DRAG_APPERTURE` as aperture has meaning elsewhere when picking entities.

Reply:
Or, `DRAG_THRESHOLD` is probably more accurate.